api/main.py

```diff
@@ -174,9 +174,23 @@ class parallelItem(BaseModel):
     co_occ: int = 0,
     limit_collection: List[str] = Query([]),
     page: int = 0,
-    # "sortmethod"
+    sort_method: str = 'position',
+
 ):
     try:
+        sort_key = ''
```

Comment:
can we refactor this a bit by extracting these few lines to an external function used like this?
```python
[sort_key, sort_direction] = get_sort_params(sort_method)
```